pkg/autoupdate/autoupdate.go

```diff
@@ -33,15 +33,25 @@ type Policy string
 const (
 	// PolicyDefault is the default policy denoting no auto updates.
 	PolicyDefault Policy = "disabled"
-	// PolicyNewImage is the policy to update as soon as there's a new image found.
-	PolicyNewImage = "image"
+	// PolicyRegistryImage is the policy to update as soon as there's a new image found.
+	PolicyRegistryImage = "image"
```

Comment:
Could you change it to "registry"?

I would love us to use this term in the future as it's less ambiguous than "image".  But (as you already did :+1:) we still need to support "image" for backwards compat.